tests/core/test_persistence.py

```diff
@@ -13,7 +13,7 @@
 from wicker.core.persistance import BasicPersistor
 from wicker.core.storage import S3PathFactory
 from wicker.schema.schema import DatasetSchema
```

Comment:
The `wicker.testing.storage.LocalStorageClass` class is used to define mock S3 functions in order to support unit tests. It certainly wasn't named very well! Let's give it a very clear name like `TestS3LocalDataStorage` instead (open to other names). Since this is an internal test class, it is ok to just change its name without worrying about backwards compatibility.